compiler/utils/astrepr.nim

```diff
@@ -122,124 +200,175 @@ proc format[I](s: set[I], sub: int): string =
 
   result.add "}"
 
-const setStyle = termFg(2, 1, 3)
+func format(i: SomeInteger): ColText = $i + style.number
+
+proc ownerChain(sym: PSym): string =
+  var own: seq[PSym] = @[sym]
+  var sym = sym
```

Comment:
'curr', for current might be better.

I think there might be an iterator for this already, I think it's hiding in AST or maybe module graph